peers/app_request_network.go

```diff
@@ -41,13 +42,22 @@ type AppRequestNetwork struct {
 
 // NewNetwork connects to a peers at the app request level.
 func NewNetwork(
-	logger logging.Logger,
+	logLevel logging.Level,
 	registerer prometheus.Registerer,
 	networkID uint32,
 	subnetIDs []ids.ID,
 	blockchainIDs []ids.ID,
 	APINodeURL string,
 ) (*AppRequestNetwork, map[ids.ID]chan message.InboundMessage, error) {
+	logger := logging.NewLogger(
+		"awm-relayer",
```

Comment:
should we differentiate that this is the network layer, and not main app

Reply:
Yup, good call